src/main/java/com/pokegoapi/util/AsyncHelper.java

```diff
@@ -0,0 +1,28 @@
+package com.pokegoapi.util;
```

Comment:
Yeah, didn't run Checkstyle yet. First prio, solve merge issue. Will fix headers right after that. Likely more is wrong/missing